cypress/integration/Switch.spec.ts

```diff
@@ -1,7 +1,7 @@
 import * as h from '../helpers';
 
 const getSwitch = () => {
-  return cy.get(`[type="checkbox"]`);
+  return cy.findByRole('checkbox');
```

Comment:
This change isn't strictly necessary, but does demonstrate it is working. I'm not sure `checkbox` is the correct role for this component yet, but I didn't update implementation.

Reply:
Yup, we should be addressing this in #387